compiler/rmi/rmi-server/src/org/jetbrains/kotlin/rmi/service/CompileServiceImpl.kt

```diff
@@ -39,54 +39,111 @@ import kotlin.concurrent.read
 import kotlin.concurrent.write
 
 
+fun nowSeconds() = System.nanoTime() / 1000000000L
+
 class CompileServiceImpl<Compiler: CLICompiler<*>>(
         val registry: Registry,
```

Comment:
What is the root of the questions? What concerns you have here?
The daemon may stay alive arbitrary long time, but current default idle timeout is 2 hours, after that it shuts itself down.